cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -238,33 +238,41 @@ RPM using Yum
 .............
 Download the Cask Yum repo definition file::
 
-  sudo curl -o /etc/yum.repos.d/cask.repo http://repository.cask.co/downloads/centos/6/x86_64/cask.repo
+  sudo curl -o /etc/yum.repos.d/cask.repo http://repository.cask.co/centos/6/x86_64/cdap/2.6/cask.repo
```

Comment:
we should use the ``|short-version|`` syntax of release 2.8 (so that we don't write "2.6" but instead put in a variable), but as long as this is not merged to develop, this is ok. If you want to add it, here it is:
```
.. container:: highlight

  .. parsed-literal::
    |$| sudo curl -o /etc/yum.repos.d/cask.repo |http:|//repository.cask.co/centos/6/x86_64/cdap/|short-version|/cask.repo
```